server.py

```diff
@@ -107,7 +104,7 @@ async def lifespan(app: FastAPI):
         }
     )
 
-app = FastAPI(lifespan=lifespan)
+app = FastAPI()
```

Comment:
Why we changed it?

Reply:
The lifespan function is async and runs in a loop which also loads the toolsets.

The new MCP async code for loading tools was causing issues within this flow , so I changed it to be sync in Holmes main function which simplifies the flow in general.

Reply:
Can you please share what was failed? Is it the new creation of event loop?